crates/solidity/inputs/language/src/dsl.rs

```diff
@@ -546,7 +546,7 @@ slang_grammar! {
 
         parser UsingDirective = ((UsingKeyword (UsingDirectivePath | UsingDirectiveDeconstruction) ForKeyword (Asterisk | TypeName) ({ introduced in "0.8.13" GlobalKeyword } ?)) terminated by Semicolon) ;
 
-        parser UsingDirectiveDeconstruction = (UsingDirectiveSymbolsList delimited by OpenBrace and CloseBrace) ;
+        parser UsingDirectiveDeconstruction = { introduced in "0.8.13" (UsingDirectiveSymbolsList delimited by OpenBrace and CloseBrace) };
```

Comment:
I wonder if we should also update `crates/solidity/inputs/language/definition/01-file-structure/04-imports/productions.yml`?

Reply:
Changed in https://github.com/NomicFoundation/slang/pull/649/commits/0691d8abbaad1e55146f4f98a9d546e6135c8b25, could you double-check if it's correct? :pray:

Reply:
left one nit. otherwise, looks great!